### PR TITLE
Tesseraify POSIX log using mixins

### PR DESCRIPTION
**Pull request by AlCutter**
This PR bring the POSIX log storage backend in #67 more fully into Tessera approach by using the Tessera mixins and storage options.

Towards #6 #13 #22

### CODE REVIEW

**Comment by mhutchinson** [comment]
Neat. I've just rebased and pushed the other PR on top of `main`.

**Comment by AlCutter** [comment]
> Neat. I've just rebased and pushed the other PR on top of `main`.

Cool, rebased this on that :)

**Comment by AlCutter** [comment]
Righto, this is up on `main` now, and ready to roll.

File: cmd/example-posix/main.go [hunk 1]
```diff
@@ -150,28 +155,32 @@ func main() {
...
+	if l := len(toAdd); l < numWorkers {
+		numWorkers = l
+	}
+
+	wg := sync.WaitGroup{}
+	for i := 0; i < numWorkers; i++ {
+		wg.Add(1)
+		go func() {
+			defer wg.Done()
+			for entry := range entries {
```
Comment (mhutchinson) [inline cmd/example-posix/main.go:173]: Can we rename `entries` to `entryChan`? It took me a confused minute to realise this threads were cooperating and not duplicating each others work by iterating over the same _slice_.

**Review by mhutchinson — APPROVED** [review]

**Pull request merged**